<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>674</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1030</width>
    <height>674</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Movie Registry</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #3B3A3D</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>1051</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
	background-color: #EF8677;
	color: white
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="studio_button">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>26</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: #0000ffff;
	
}

QPushButton:pressed
{
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>pictures_for_project/flag_icon-icons.com_69377.png</normaloff>pictures_for_project/flag_icon-icons.com_69377.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="show_movie">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: #0000ffff;
	
}

QPushButton:pressed
{
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>pictures_for_project/database_refresh_icon_137697.png</normaloff>pictures_for_project/database_refresh_icon_137697.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>66</height>
      </size>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="movie_label">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>96</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: black;
    qproperty-alignment: 'AlignVCenter | AlignLeft';
 	qproperty-wordWrap: true;
}</string>
     </property>
     <property name="text">
      <string>Обновить БД</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="studio_label">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>96</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>UpArrowCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: black;
    qproperty-alignment: 'AlignVCenter | AlignLeft';
 	qproperty-wordWrap: true;
}</string>
     </property>
     <property name="text">
      <string>По стране</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_position">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: #0000ffff;
	
}

QPushButton:pressed
{
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>pictures_for_project/tresh.png</normaloff>pictures_for_project/tresh.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="movie_search">
     <property name="geometry">
      <rect>
       <x>394</x>
       <y>26</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: #0000ffff;
	
}

QPushButton:pressed
{
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>pictures_for_project/lupa.png</normaloff>pictures_for_project/lupa.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="delete_label">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>96</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>UpArrowCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: black;
    qproperty-alignment: 'AlignVCenter | AlignLeft';
 	qproperty-wordWrap: true;
}</string>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="search_label">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>96</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: black;
    qproperty-alignment: 'AlignVCenter | AlignLeft';
 	qproperty-wordWrap: true;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>По названию</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="movie_add">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: #0000ffff;
	
}

QPushButton:pressed
{
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>pictures_for_project/add_film.png</normaloff>pictures_for_project/add_film.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="add_label">
     <property name="geometry">
      <rect>
       <x>196</x>
       <y>96</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: black;
    qproperty-alignment: 'AlignVCenter | AlignLeft';
 	qproperty-wordWrap: true;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="year_button">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>30</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: #0000ffff;
	
}

QPushButton:pressed
{
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>pictures_for_project/img_523733.png</normaloff>pictures_for_project/img_523733.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="year_label">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>96</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>UpArrowCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: black;
    qproperty-alignment: 'AlignVCenter | AlignLeft';
 	qproperty-wordWrap: true;
}</string>
     </property>
     <property name="text">
      <string>По году издания</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="genre_button">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: #0000ffff;
	
}

QPushButton:pressed
{
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>pictures_for_project/img_63405.png</normaloff>pictures_for_project/img_63405.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="genre_label">
     <property name="geometry">
      <rect>
       <x>726</x>
       <y>96</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>UpArrowCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: black;
    qproperty-alignment: 'AlignVCenter | AlignLeft';
 	qproperty-wordWrap: true;
}</string>
     </property>
     <property name="text">
      <string>По жанру</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	background-color: #3B3A3D;
	color: white;
	border-color: #3B3A3D;
}</string>
     </property>
     <property name="text">
      <string>Поиск по базе</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	background-color: #3B3A3D;
	color: white;
	border-color: #3B3A3D;
}</string>
     </property>
     <property name="text">
      <string>Редактирование</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	background-color: #3B3A3D;
	color: white;
	border-color: #3B3A3D;
}</string>
     </property>
     <property name="text">
      <string>Обновление</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QPushButton" name="excel_button">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>30</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: #0000ffff;
	
}

QPushButton:pressed
{
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>pictures_for_project/732089.svg</normaloff>pictures_for_project/732089.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>10</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	background-color: #3B3A3D;
	color: white;
	border-color: #3B3A3D;
}</string>
     </property>
     <property name="text">
      <string>Эксель</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="excel_label">
     <property name="geometry">
      <rect>
       <x>828</x>
       <y>96</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>UpArrowCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: black;
    qproperty-alignment: 'AlignVCenter | AlignLeft';
 	qproperty-wordWrap: true;
}</string>
     </property>
     <property name="text">
      <string>Импорт в эксель</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="excel_button_top">
     <property name="geometry">
      <rect>
       <x>956</x>
       <y>30</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: #0000ffff;
	
}

QPushButton:pressed
{
	background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>pictures_for_project/732089.svg</normaloff>pictures_for_project/732089.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="excel_label_top">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>96</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>UpArrowCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: black;
    qproperty-alignment: 'AlignVCenter | AlignLeft';
 	qproperty-wordWrap: true;
}</string>
     </property>
     <property name="text">
      <string>TOP 250</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>991</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #EF8677;
gridline-color: rgb(0, 0, 0);</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Фильм</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Год</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Страна</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Жанр</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Режисер</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Длительность</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ограничение</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Просмотры</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>580</y>
      <width>921</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Поиск:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
